src/instruction_handlers/precompiles.rs

```diff
@@ -50,7 +50,7 @@ fn precompile_call(
         let query = LogQuery {
             timestamp: Timestamp(0),
             key: abi.to_u256(),
-            tx_number_in_block: Default::default(),
```

Comment:
Please don't do this, add a comment instead that says that only the first fields are read by the precompile